### ISSUE TITLE
Request to add Duet Display

### ISSUE THREAD

**Issue by AlJohri** [microsoft/winget-pkgs#5666]
# Package Requested

Duet Display
https://www.duetdisplay.com/
`https://updates.duetdisplay.com/latestWindows` -> `https://duet.nyc3.cdn.digitaloceanspaces.com/Windows/1_9/DuetSetup-1-9-2-7.exe`

```
PS C:\Users\aljoh\Downloads> Get-FileHash -Path .\DuetSetup-1-9-2-7.exe -Algorithm SHA256

Algorithm       Hash                                                                   Path
---------       ----                                                                   ----
SHA256          0DEB24426F8FF8C161F96988C4FB9DA857232A61D21881382C93EE579A7B9515       C:\Users\aljoh\Downloads\DuetSetup-1-9-2-7.exe
```

This one seems like a non-standard exe? I tried passing `/?` and got these results:

![image](https://user-images.githubusercontent.com/2790092/102886861-e6b38180-4423-11eb-9f44-7c35f909813e.png)

It mentions `msiexec` though so I imagine it should be compatible?

Manifest:

Fill out as much of the manifest as you can:

```YAML
Id: Kairos.DuetDisplay
Publisher: Kairos
Name: DuetDisplay
Version: 1.9.2.7
License: string # the open source license or copyright
InstallerType: string # enumeration of supported installer types (exe, msi, msix, inno, wix, nullsoft, appx)
Installers:
  - Arch: string # enumeration of supported architectures
    Url: https://duet.nyc3.cdn.digitaloceanspaces.com/Windows/1_9/DuetSetup-1-9-2-7.exe
    Sha256: 0DEB24426F8FF8C161F96988C4FB9DA857232A61D21881382C93EE579A7B9515
# ManifestVersion: 0.1.0
```

**Comment by AlJohri** [comment]
I reached out to Duet support and they provided the following instructions. Looks like they have an MSI version which should make this much simpler. However, I don't see a public url with the MSI installer at this time. Reached out to see if they can provide one.

>Here are the steps on how to install Duet using the CMD:
>
>1. Delete all files of Duet from your computer, leave it completely clean.
>2. Download Duet 1.9.1.9 MSI: https://slack-files.com/T03JN9WJD-F013PHHGR7D-1a156b1477
>3. The MSI version should be on the "Downloads" folder of your computer. Please don't move it.
>4. On your keyboard press Win + R keys > "The Run" pop-up will open > Type "cmd" > Press "Enter" key, the command prompt will open.
>5. Please type the following text on the command prompt: msiexec /i "C:\Users\(Your PC Name)\Downloads\DuetSetup-1-9-1-9.msi" /qn
>6. And follow the installation prompts of Duet to finish the process.

**Comment by AlJohri** [comment]
Duet says a public url for the MSI does not exist.

>I'm afraid there's no public link with a MSI installed with Duet as our development team does not create an MSI for each version - the last MSI version that is available is for v1.9.1.9 which my colleague instructed how to get it installed properly.

Closing this ticket until Duet adds a public MSI url.

**Comment by lheinke** [comment]
Hi @AlJohri. I just created a pull request (#9652) for a Duet Display manifest. I extracted the msi from the exe installer boostrapper  to see which `msioptions` are available. The `/quiet` flag makes the whole bootstrapper quiet and by setting the common flag `/norestart` unwanted restarts are prevented. Thought you might still be interested in an installation via winget.
